### DISCUSSION TITLE
Count device types - DONE

### DISCUSSION THREAD

**Discussion by dannyvankooten** [ibericode/koko-analytics#219] [Ideas]
Regarding device types, which types of devices should we group together? 

Maybe screen resolution is a better metric that actually influences our design decisions? 

Maybe we can group on certain screen widths, as that is what we usually base responsive design cutoff points for?

**Comment by fancsali** [comment]
I'd certainly still keep this along the lines of mobile-tablet-desktop.

But it does make sense to go by screen resolution. Perhaps a bit of research could tell us what the usual breakpoints for these three categories are (today), and then go by that...

**Comment by BiancaNL** [comment]
Good question, I don't know actually. Some thoughts: Tablet does [not seem to be used](https://gs.statcounter.com/platform-market-share/desktop-mobile-tablet) that much. Media breakpoints can be different depending on the design. I think screen sizes do make more sense these days as responsive design is also evolving.

**Comment by dannyvankooten** [comment]
![image](https://github.com/ibericode/koko-analytics/assets/885856/5552f6de-dd00-4398-a60e-3ddce5eada3b)

Using [Koko Analytics Pro](https://www.kokoanalytics.com/pricing/) you can utilize the event tracking feature to track screen resolutions and browser languages too. 

```js
koko_analytics.trackEvent('Screen resolution', screen.width + ' x ' + screen.height);
koko_analytics.trackEvent('Browser language', navigator.language);
```

This then renders like this in the dashboard:

![image](https://github.com/ibericode/koko-analytics/assets/885856/0d7d6b69-1fa7-4154-ac64-9cbc0ef818f6)

Perhaps we should build on this and include extra metrics like screen resolution, device type, browser language, operating system etc. as common events that can easily be toggled on or off?

**Reply by BiancaNL** [reply]
Works for me 😀

**Comment by dannyvankooten** [comment]
This was implemented in Koko Analytics Pro version 2.2, released on Jan 21, 2026.

https://pro.kokoanalytics.com/plugin/changelog

It allows you to keep track of browsers, operating systems and device types.

<img width="896" height="238" alt="image" src="https://github.com/user-attachments/assets/b4fd2cda-0acb-48fa-ac17-fe38f63109d3" />
